data.js

```diff
@@ -963,5 +972,44 @@ if (0) {
 			}
 		}
 		return b.join('');
-	}
+	},
+  makeRtkHtml: function (idxHeisig){
+    var  str="",text = rcxMain.config.rtkStories[idxHeisig][5];  
+    
+    res = text.match(/\*(.*?)\*/g);
+    if (res != null){
+      for (var i = 0; i < res.length; i++) {
+        if (res[i]!=""){
+          //eliminar "*"
+          var x  = "<em>"+res[i].replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "")+"</em>";
+          // reemplazamos los *xxxx* que haya por el valor de x
```

Comment:
this comment could be more specific it seems like maybe it's from a recipe but it'd be good to know what exactly you're replacing.